hbase-migration-tools/mirroring-client/bigtable-hbase-mirroring-client-2.x-parent/bigtable-hbase-mirroring-client-1.x-2.x-integration-tests/pom.xml

```diff
@@ -262,7 +262,8 @@ limitations under the License.
     <dependency>
       <groupId>io.prometheus</groupId>
```

Comment:
where is this used?

Reply:
Its used in a rule for tests metrics export:
https://github.com/googleapis/java-bigtable-hbase/blob/ff97ac926c00aa3c09f4ed9ceba6a017d898e695/hbase-migration-tools/mirroring-client/bigtable-hbase-mirroring-client-1.x-parent/bigtable-hbase-mirroring-client-1.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/mirroring/utils/PrometheusStatsCollectionRule.java#L42-L45

I'm not entirely sure where that rule is actually used, but for now I just wanna stick to dependency handling as much as possible